ansible/golden_config_db/smartswitch_t1-28-lag.json

```diff
@@ -0,0 +1,111 @@
+{
+    "CHASSIS_MODULE": {
+        "DPU0": {
+            "admin_status": "down"
+        },
+        "DPU1": {
+            "admin_status": "down"
+        },
+        "DPU2": {
+            "admin_status": "down"
+        },
+        "DPU3": {
+            "admin_status": "down"
+        },
+        "DPU4": {
+            "admin_status": "down"
+        },
+        "DPU5": {
+            "admin_status": "down"
+        },
+        "DPU6": {
+            "admin_status": "down"
+        },
+        "DPU7": {
+            "admin_status": "down"
+        }
+    },
+    "DHCP_SERVER_IPV4": {
+        "bridge-midplane": {
+            "gateway": "169.254.200.254",
+            "lease_time": "600",
```

Comment:
I think for lab devices we can set an infinite lease time, otherwise it might expire frequently. See https://github.com/sonic-net/SONiC/blob/master/doc/smart-switch/ip-address-assigment/smart-switch-ip-address-assignment.md#npu-platformjson

Reply:
Hi @prabhataravind , this will require YANG model update.
Currently YANG doesn't support `inifinite` string in lease_time